### PR TITLE
Add feature to read chaos parameters from aws ssm

### PR DESCRIPTION
**Pull request by brignolij**
- add the feature to read chaos settings from aws ssm instead of using stream url parameters.
- add lambdav2 to be able to run from aws lambda
- update Dockerfile to make it smaller.

### CODE REVIEW

**Comment by birme** [comment]
Awesome stuff! Thanks!
If you could just rebase with master and squash the PR commits I will have a look. I think perhaps the lambdav2 implementation could replace the existing one

**Comment by brignolij** [comment]
@birme thanks for your feedback.
about lambdav2, i did it because i was not able to make it run using "v1".
Maybe do you know if still working using v1 ? i can replace with v2, at your convenience.

**Comment by birme** [comment]
> @birme thanks for your feedback.
> about lambdav2, i did it because i was not able to make it run using "v1".
> Maybe do you know if still working using v1 ? i can replace with v2, at your convenience.

I suggest you replace v1 with v2, it looked good!

**Comment by brignolij** [comment]
@birme done ;)

File: DockerfileLambda [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM public.ecr.aws/lambda/nodejs:16 as builder
```
[the remark this answers is not in the document]
Comment (brignolij) [reply DockerfileLambda]: @birme done

**Comment by birme** [comment]
Again, thanks for the contribution. I see that the linting, prettier and type check fails. It would be great if you could address these.

**Comment by ShibireX** [comment]
Hey! Cool stuff. I'm having some trouble getting your project to run, could you perhaps add some (more) instructions as to how to get it running locally and manage the corruptions?

**Comment by brignolij** [comment]
@birme  @ShibireX build issue is caused by demo.ts, I don't understand the purpose of this file , should i delete it ?

**Comment by birme** [comment]
> @birme @ShibireX build issue is caused by demo.ts, I don't understand the purpose of this file , should i delete it ?

I'd rather see to adapt it to using the new lambda implementation instead. If possible. Thanks!

**Comment by birme** [comment]
Thanks, you're almost there :)
Now just fix the lint and prettier issues. You can run it locally
```
npm run lint
```

and 

```
npm run pretty
```

**Comment by birme** [comment]
I get a compilation error now for some reason?

```
src/lambda.ts:4:22 - error TS2304: Cannot find name 'awsLambdaFastify'.

4 export const proxy = awsLambdaFastify(app);
```

**Comment by birme** [comment]
Nice, thank you for this! Will merge it

**Pull request merged**